gavel/templates/welcome.html

```diff
@@ -1,20 +1,68 @@
-{% extends "layout.html" %}
-{% block title %}Welcome{% endblock %}
-{% block content %}
+{% extends "layout.html" %} {% block title %}Welcome{% endblock %} {% block
+content %}
 <div class="item">
   <div class="banner">
     <h1>Welcome</h1>
   </div>
-  <div class="info">
-    {{ content }}
-  </div>
+  <div class="info">{{ content }}</div>
 </div>
 
 <div class="item">
+  <div class="banner">
+    <h1>Welcome</h1>
+  </div>
+  <div class="info">
+    {{ content }}
```

Comment:
The welcome page content is being loaded dynamically, consider loading the "how to use Gavel" dynamically too, or removing "content" from it, as it currently loads the layout wrongly

Desktop
![image](https://github.com/user-attachments/assets/14536c9e-82ef-4b28-9f8e-dc1c9c7081b5)

Mobile
![image](https://github.com/user-attachments/assets/fca4ad3b-a2ac-4a83-9735-d22836faa0ba)

Reply:
@TungNguyen12

Reply:
I removed this the duplicate content: <div class="item">
  <div class="banner">
    <h1>Welcome</h1>
  </div>
  <div class="info">
    {{ content }}
  </div>
  <div class="info">{{ content }}</div>
</div>